site.js

```diff
@@ -61,26 +87,32 @@ geocoder.on('select', function(data) {
     if (searchMarker) {
         map.removeLayer(searchMarker);
     }
-    this._closeIfOpen();
+    this._closeIfOpen(); //'this' is the geocoder search bar on the page
     searchMarker = L.marker(data.feature.center.reverse());
     searchMarker.bindPopup(data.feature.place_name);
     searchMarker.addTo(map);
 });
 
+
+/**
+* Onload callbacks for buffers and bikelanes
+*/
+
+//Styles the buffer using bufferStyle variable, and 
+function loadBuffer(data) {
+    var buffer = data.target;
+    setProperties(buffer.getGeoJSON());
+    //buffer.setGeoJSON(buffer.getGeoJSON())
+}
+
 // Each buffer feature object needs to have the properties set individually
 function setProperties(buffer) {
     for (var i = 0; i < buffer.features.length; i++) {
         buffer.features[i].properties = bufferStyle;
     }
 }
 
-// onload callbacks for buffers and bikelanes
```

Comment:
I'll fix that - it's not actually deleted, but rather I moved it  below the loadBuffer function since loadBuffer is the 'primary' function and setProperties is only ever used as a sub-function of loadBuffer. The real error is accidentally leaving line 105 commented out (I was trying to understand how it was working). Sorry for introducing a bug, thanks for checking!